bin/node-pg-migrate

```diff
@@ -260,6 +260,9 @@ function readTsconfig() {
       // eslint-disable-next-line security/detect-non-literal-fs-filename
       const config = fs.readFileSync(path.resolve(process.cwd(), tsconfigPath), { encoding: 'utf-8' })
       tsconfig = json5 ? json5.parse(config) : JSON.parse(config)
+      if (tsconfig['ts-node']) {
+        tsconfig = { ...tsconfig, ...tsconfig['ts-node'] }
```

Comment:
Is fully overriding `compilerOptions` also wanted? Or should it be more like a merging?

Reply:
No, in this context it is not 🙂
It would fully override the compilerOptions instead of merging them together 
What we might need here is a deepMerge functionality 
Unless this is specifically by design